x/distribution/keeper/key.go

```diff
@@ -0,0 +1,49 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// keys/key-prefixes
+var (
+	FeePoolKey               = []byte{0x00} // key for global distribution state
+	ValidatorDistInfoKey     = []byte{0x01} // prefix for each key to a validator distribution
+	DelegationDistInfoKey    = []byte{0x02} // prefix for each key to a delegation distribution
+	DelegatorWithdrawInfoKey = []byte{0x03} // prefix for each key to a delegator withdraw info
+
+	// transient
+	ProposerKey              = []byte{0x00} // key for storing the proposer operator address
```

Comment:
Ideally we could call them `TKey` or something to disambiguate

Reply:
for transient store? yeah sure - makes sense!

Reply:
irrelevant now tstore removed